components/script/task_source/mod.rs

```diff
@@ -38,6 +39,7 @@ pub enum TaskSourceName {
     MediaElement,
     Websocket,
     Timer,
+    Gamepad,
```

Comment:
https://w3c.github.io/gamepad/#dfn-gamepad-task-source

(even if it wasn't done for the others...)